cpu/stm32w108/hal/micro/cortexm3/mfg-token.h

```diff
@@ -33,32 +36,28 @@
 /**
  * \brief Macro for translating token definitions into size variables.
  * This provides a convenience for abstracting the 'sizeof(type)' anywhere.
- *
- * \param name: The name of the token.
- *
- * \param type: The token type.  The types are found in token-stack.h.
  */
```

Comment:
Maybe it's better to just remove TOKEN_MFG from Doxygen, but keep the documentation. Without understanding the code the documentation is not helpful. And explaining what's happening is IMO not worth the trouble.